### ISSUE TITLE
OAC indexing failures to investigate

### ISSUE THREAD

**Issue by corylown** [sul-dlss/stanford-arclight#637]
```
just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m1930.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2163.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/pc0224.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m1900.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2757.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/0900earth.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/photo0674.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1496.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2280.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1580.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2651.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1581.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2297.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2505.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2781.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1506.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/a0141.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2088.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2553.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1296.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2559.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2706.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m0319.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2568.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	just now 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1513.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:
 

22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2695.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1488.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2055.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2607.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m1925.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2633.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1358.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1616.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/photo0631.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/scm0484.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2139.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2740.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2705.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/photo0402.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2555.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2686.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1593.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1525.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2694.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2810.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2005.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1609.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1604.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2587.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	22 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2753.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:
 

32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2685.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1519.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2043.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2751.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2746.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1572.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1414.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2675.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1582.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2496.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2606.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/photo0617.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc1297.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/uarc/sc0983.xml", "arclight_repository_code"=>"uarc"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2645.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2558.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2713.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:

	32 seconds ago 	default 	IndexEadJob 	
{"file_path"=>"/opt/app/arclight/data/manuscripts/m2052.xml", "arclight_repository_code"=>"manuscripts"}
	IndexEadJob::IndexEadError:
 
```

**Comment by marlo-longley** [comment]
@corylown -- looking at the first ones here, they are all missing EADID. 
Trying index for example https://voro.cdlib.org/oac-ead/prime2002/stanford/mss/m2297.xml locally, I get: 

```
bundler: failed to load command: traject (/Users/mlongley/.rbenv/versions/3.3.0/bin/traject)
/Users/mlongley/.rbenv/versions/3.3.0/lib/ruby/gems/3.3.0/gems/arclight-1.1.4/lib/arclight/normalized_id.rb:25:in `normalize': id must be present for all documents and components (Arclight::Exceptions::IDNotFound)
```

I then tried to index this and similar files on my `collection-id` branch with the custom normalizer -- no failures there.

**Comment by corylown** [comment]
May be fixed by #634

**Comment by corylown** [comment]
Confirmed that #634 has resolved this issue and these finding aids are now indexed in production.
